cpp/include/cudf/datetime.hpp

```diff
@@ -206,6 +206,18 @@ std::unique_ptr<cudf::column> is_leap_year(
   cudf::column_view const& column,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * `column[i]`. It will be null` if the input row at `column[i]` is null.
```

Comment:
Wait, it seems that you have accidentally removed the beginning part of this doxygen.

Reply:
oops my bad, fixed now